packages/internals/src/engine-commands/getConfig.ts

```diff
@@ -1,30 +1,17 @@
 import Debug from '@prisma/debug'
-import { getCliQueryEngineBinaryType } from '@prisma/engines'
-import { BinaryType } from '@prisma/fetch-engine'
 import type { DataSource, EnvValue, GeneratorConfig } from '@prisma/generator-helper'
 import chalk from 'chalk'
-import execa from 'execa'
 import * as E from 'fp-ts/Either'
 import { identity, pipe } from 'fp-ts/lib/function'
-import * as TE from 'fp-ts/TaskEither'
 import { match, P } from 'ts-pattern'
 
-import { ErrorArea, isExecaErrorCausedByRustPanic, isWasmPanic, RustPanic } from '../panic'
+import { ErrorArea, isWasmPanic, RustPanic } from '../panic'
 import { prismaFmt } from '../wasm'
 import { addVersionDetailsToErrorMessage } from './errorHelpers'
-import {
-  createDebugErrorType,
-  createSchemaValidationError,
-  loadNodeAPILibrary,
-  preliminaryBinaryPipeline,
-  preliminaryNodeAPIPipeline,
-  unlinkTempDatamodelPath,
```

Comment:
Are these utils still used anywhere?

Reply:
Still used in `getDmmf.ts`, will probably be gone before the sprint is over (as `getDmmf` will be ported to Wasm)